XRTK.SDK/Packages/com.xrtk.sdk/Editor/SDKPackageInstaller.cs

```diff
@@ -36,15 +37,23 @@ private static void ImportPackageAssets()
 
         private static void CheckPackage()
         {
+            var updateGUIDs = false;
 
             if (!EditorPreferences.Get($"{nameof(SDKPackageInstaller)}.Profiles", false))
             {
-                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Profiles", PackageInstaller.TryInstallAssets(HiddenProfilePath, $"{DefaultPath}\\Profiles"));
+                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Profiles", PackageInstaller.TryInstallAssets(HiddenProfilePath, $"{DefaultPath}\\Profiles", false));
+                updateGUIDs = true;
```

Comment:
should we break out the TryInstallAssets to set the flag, then pass the flag into the pref setter?

I'm not sure we always want to regenerate guids if installation fails for any reason.

Reply:
Updated, but still not sure I'm happy with it

Reply:
Yeah I agree. Wanna play with it some more and see what you think?